packages/block-editor/src/components/inserter/index.js

```diff
@@ -92,7 +92,6 @@ class Inserter extends Component {
 				position={ position }
 				onToggle={ this.onToggle }
 				expandOnMobile
-				headerTitle={ title }
```

Comment:
FWIW, this was our only usage of `headerTitle`. It's now dead code in `Inserter`. That said, there's a compatibility question of removing it, and I suppose there may be valid use-cases for it in the future.